map_test.go

```diff
@@ -2275,17 +2286,17 @@ func ExampleMap_NextKey() {
 		panic(err)
 	}
 
-	var firstKey string
-	if err := hash.NextKey(nil, &firstKey); err != nil {
+	keys := make([]string, 2)
+	if err := hash.NextKey(nil, &keys[0]); err != nil {
 		panic(err)
 	}
 
-	var nextKey string
-	if err := hash.NextKey(firstKey, &nextKey); err != nil {
+	if err := hash.NextKey(keys[0], &keys[1]); err != nil {
 		panic(err)
 	}
-
-	// Order of keys is non-deterministic due to randomized map seed
```

Comment:
Ack on the sort, but don't remove the comment.

Reply:
I also discovered this is possible:

```
// Unordered output:
```